src/utils/optionsparser.h

```diff
@@ -116,11 +116,14 @@ class OptionsParser {
   void AddContext(const std::string&);
 
  private:
+  // Print help to std::cerr
```

Comment:
`Prints` (it's 3rd person, describing what function does).
Also, period in the end of the sentence. :)